components/backup-stream/src/endpoint.rs

```diff
@@ -655,7 +655,6 @@ where
                         .try_for_each(|r| {
                             tx.blocking_send(ObserveOp::Start {
                                 region: r.region.clone(),
-                                handle: ObserveHandle::new(),
```

Comment:
If remove the handle here, will some stale region change of Lock CF came and Lock forever? 
@YuJuncen WDYT?

Reply:
It seems no problem, since every places is use `equal to` condition rather than `less` or `greater` to compare handle id.